OMCompiler/Compiler/BackEnd/BackendDAEUtil.mo

```diff
@@ -6113,7 +5854,7 @@ algorithm
       tuple<BackendDAE.Variables, Boolean, Boolean, Integer, array<Integer>, list<Integer>> tpl;
 
     case (DAE.LUNARY(exp=e1), (vars, bs, isInitial, it, at, pa)) equation
-      (_, (vars, _, _, _, _, pa)) = Expression.traverseExpTopDown(e1, traversingAdjacencyRowExpSolvableEnhancedFinder, (vars, true, isInitial, it, at, pa));
+      (_, (vars, _, _, _, _, pa)) = Expression.traverseExpTopDown(e1, traversingAdjacencyRowExpSolvableEnhancedFinder, (vars, bs, isInitial, it, at, pa));
```

Comment:
Somehow this did not make it in the last review so here is just a single comment:

Why did you change `true` to `bs`? What does `bs` stand for? Do we need to change this in all function calls below?

Reply:
Ya I know :( I wanted to change it but it will clutter things in this commit. 

The variable signifies "unsolvable". If it is true it means the exp is unsolvable. So as we traverse down we will mark all vars we find as unsolvable. 

It was set to true here which meant it was marking the vars in the exp of the LUNARY() all unsolvable. 

I would like to change it to at least say "solvable" and switch the boolean values around.  The best way would be to create a record instead of passing a tuple and match is with names like "bs".

Reply:
Ah i see, so when i enter let's say a function call dependency list, i mark all occurring variables with unsolvable.

Reply:
Yes. I think so.